sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ReadConsistencyStrategy.java

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.ImplementationBridgeHelpers;
+import com.azure.cosmos.util.Beta;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Represents the read consistency strategies supported by the Azure Cosmos DB service.
+ * <p>
+ * The requested read consistency strategy can be chosen independent of the consistency level
+ * provisioned for the database account.
+ * <p>
+ * NOTE: The ReadConsistencyStrategy is currently only working when using direct mode
+ */
+@Beta(value = Beta.SinceVersion.V4_69_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
```

Comment:
For this, are we not going to talk about how it overrides `ConsistencyLevel` too?

Reply:
No need to hide. Added one sectione explaining this.